src/style.css

```diff
@@ -1,3 +1,95 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  height: 60px;
+}
+
+a {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  height: 60px;
```

Comment:
Avoid styling elements using tag names (except for 'html' and 'body'). It's better to use classes to apply styles to elements to ensure that your styling is as specific as possible and doesn't unintentionally affect other elements.